mysql-test/suite/sys_vars/t/sysvars_innodb.test

```diff
@@ -5,13 +5,13 @@
 
 --vertical_results
 --replace_regex /^\/\S+/PATH/ /\.\//PATH/
+--replace_result 'resides in persistent memory or ' ''
```

Comment:
Alright, I see this unifies the variation in the help text:

```
   --innodb-log-file-mmap 
-                      Whether ib_logfile0 resides in persistent memory or
-                      should initially be memory-mapped
+                      Whether ib_logfile0 should initially be memory-mapped
                       (Defaults to on; use --skip-innodb-log-file-mmap to disable.)
```

Reply:
Typically places that have documentation to users show one version per software version. For example https://manpages.debian.org/unstable/mariadb-server-core/mariadbd.8.en.html has 3 versions for the 3 versions of MariaDB in Debian currently. Now it seems that the text will be different depending on architecture, so documentation should maybe have its own version per architecture.. obviously that would not make sense in any online collection of documentation, but for binaries outputting different help texts on different platforms it is less clear that may be the case